src/lib/blockdev.c.in

```diff
@@ -41,7 +41,7 @@
  *
  */
 
-#define DEFAULT_CONF_DIR_PATH "/etc/libblockdev/conf.d/"
+#define DEFAULT_CONF_DIR_PATH "/etc/libblockdev/@MAJOR_VER@/conf.d/"
```

Comment:
Btw. this should follow the `libbdconfdir` or `sysconfdir` configured prefix variable, just like every other file in the automake project.